client/my-sites/site-settings/reading-newsletter-settings/EmailsTextSetting.tsx

```diff
@@ -36,32 +40,44 @@ export const EmailsTextSetting = ( { value, disabled, updateFields }: EmailsText
 		};
 
 	return (
-		<div className="site-settings__emails-test-settings-container">
+		<div className="site-settings__emails-text-settings-container">
 			<FormFieldset>
 				{ /* @ts-expect-error FormLegend is not typed and is causing errors */ }
 				<FormLegend>
 					These settings change the emails sent from your site to your readers
 				</FormLegend>
-				<FormLabel htmlFor="welcome_email_text">{ translate( 'Welcome email text' ) }</FormLabel>
+				<FormLabel htmlFor="confirmation_email_message">
+					{ hasTranslation( 'Confirmation email message' ) || locale.startsWith( 'en' )
```

Comment:
Question. `locale.startsWith('en')` is just to cover other en languages right, like `en-GB`?

Reply:
Yes, that's correct. I hadn't used it in my previous PRs that added language fallback, but then I saw the usage of `locale.startsWith('en')` in one of @mashikag PRs.

What I prefer about it is that we can make the new copy available to English speakers earlier.